repo/packages/S/spark/0/marathon.json

```diff
@@ -0,0 +1,38 @@
+{
+    "id": "{{spark.framework-name}}",
+    "cpus": {{spark.cpus}},
+    "mem": {{spark.mem}},
+    "instances": {{spark.instances}},
+    "cmd": "echo spark.mesos.principal={{mantl.mesos.principal}} >> ./conf/spark-defaults.conf && echo spark.mesos.secret={{mantl.mesos.secret}} >> ./conf/spark-defaults.conf && mv ./conf/log4j-dispatcher.properties ./conf/log4j.properties && ./bin/spark-class org.apache.spark.deploy.mesos.MesosClusterDispatcher --port $PORT0 --webui-port $PORT1 --master {{mesos.master}} --zk {{spark.zookeeper}} --host {{spark.host}} --name {{spark.framework-name}}",
```

Comment:
Sure! I fixed it, and tested it against my authentication enabled cluster. It worked great. You might want to try it on some cluster with disabled authentication too.